index.js

```diff
@@ -10,40 +11,64 @@ const server = express();
 server.use(express.json());
 server.use(cors());
 
-server.use(
-    session({
-        name: "session1",
-        secret: "placeholder",
-        cookie: {
-            maxAge: 1 * 24 * 60 * 60 * 1000
-        },
-        httpOnly: true,
-        resave: false,
-        saveUninitialized: false
-    })
-);
+function generateToken(user) {
+    const payload = {
+        username: user.username,
+        department: user.department
+    };
+
+    const secret = 'secretsecretsarenofunsecretsecretshurtsomeone';
```

Comment:
Try sticking this in a `.env` file so that malicious folks will never see it! The setup is fairly easy, you just have to require the package once, say at the top of this file, and call it's config method:
```javascript
require('dotenv').config();
// ...
const secret = process.env.JWT_SECRET 
  || 'secretsecretsarenofunsecretsecretshurtsomeone';
```
in your .env:
```javascript
JWT_SECRET=somesecretwewillneverknow
```
and be sure to add `.env` into your `.gitignore`